packages/api/src/validators/attendee.ts

```diff
@@ -0,0 +1,13 @@
+import { EventResponseStatus } from "@agreeto/db";
```

Comment:
Do you like to organise this in different folders or can we colocate the validators a bit closer to the actual feature?